include/exadg/postprocessor/lift_and_drag_calculation.cpp

```diff
@@ -101,6 +122,13 @@ void
   Force = dealii::Utilities::MPI::sum(Force, mpi_comm);
 }
 
+void
+LiftAndDragData::print(dealii::ConditionalOStream & pcout, bool const unsteady) const
+{
+  free_print(pcout, unsteady, *this);
```

Comment:
why is this realized like this and not like it is done for other modules?

Reply:
I will change it to be consistent. I am just used to have free prints instead of members and didn't think about the ExaDG conventions during implementation.

Reply:
[e482ac8](https://github.com/exadg/exadg/pull/262/commits/e482ac8101da49a4f0c27e76d4da9a75f683a836)